src/styles/respec.css.js

```diff
@@ -23,13 +23,7 @@ export default css`
 }
 
 /* --- INLINES --- */
-h1 abbr,
-h2 abbr,
-h3 abbr,
-h4 abbr,
-h5 abbr,
-h6 abbr,
-a abbr {
+:is(h1, h2, h3, h4, h5, h6, a) abbr {
```

Comment:
:shipit: 😍

Reply:
heh, let's hope the CSS validator gets fixed soon... 
FYI... read https://web.dev/css-is-and-where/